src/Orleans/Statistics/RuntimeStatisticsGroup.cs

```diff
@@ -12,16 +12,16 @@ internal class RuntimeStatisticsGroup
     {
         private static readonly TraceLogger logger = TraceLogger.GetLogger("RuntimeStatisticsGroup", TraceLogger.LoggerType.Runtime);
 
-        private PerformanceCounter cpuCounter;
-        private PerformanceCounter availableMemoryCounter;
+        private PerformanceCounter cpuCounterPF;
```

Comment:
Yes, PerformanceCounter.
What way it is easy to see which counter is coming from PerformanceCounter and which not, and can make sure we check each PerformanceCounter for null.